### PR TITLE
Adds elk feature that contains all boms

### PR DESCRIPTION
**Pull request by m4rkmckenna**

### CODE REVIEW

**Review by googlielmo — APPROVED** [review]
LGTM

File: feature/pom.xml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>brooklyn-elk</artifactId>
+        <groupId>io.brooklyn.elk</groupId>
+        <version>1.0.0</version>
```
Comment (googlielmo) [inline feature/pom.xml]: should this be 1.0.1-SNAPSHOT?

**Comment by googlielmo** [comment]
@m4rkmckenna I added just one comment, but okay to merge

File: feature/pom.xml (hunk 1 above)
Comment (m4rkmckenna) [reply feature/pom.xml]: It should i will change

**Pull request merged**